bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/TeslaChannelSelectorProxy.java

```diff
@@ -45,6 +45,48 @@ public class TeslaChannelSelectorProxy {
     public enum TeslaChannelSelector {
 
         API("api_version", "api", DecimalType.class, true),
+        AR_DESTINATION("active_route_destination", "destinationname", StringType.class, true),
+        AR_LATITUDE("active_route_latitude", "ar_location", DecimalType.class, false) {
```

Comment:
```suggestion
        AR_LATITUDE("active_route_latitude", "destinationlocation", DecimalType.class, false) {
```

I think, but I'm not sure. This (already existing) enum logic seems a bit obscure to me.